File: bot.js
const Discord = require('discord.js'); // Подключаем библиотеку discord.js
const robot = new Discord.Client(); // Объявляем, что robot - бот
const fs = require("fs");
const comms = require("./comms.js"); // Подключаем файл с командами для бота
let config = require('./config.json'); // Подключаем файл с конфигруацией

const prefix = config.prefix; // «Вытаскиваем» из него префикс
const token = process.env.token;

robot.on("ready", function () {

  /* При успешном запуске, в консоли появится сообщение «[Имя бота] запустился!» */
  console.log(robot.user.username + " запустился!");
});


robot.on('message', (msg) => { // Реагирование на сообщения

  if (msg.author.username != robot.user.username && msg.author.discriminator != robot.user.discriminator) {
    var comm = msg.content.trim() + " ";
    var comm_name = comm.slice(0, comm.indexOf(" "));

    var args = comm.split(" ");

    if (args[0] == "!restart") {
      restart(msg, args);
    } else {
      for (comm_count in comms.comms) {
        var comm2 = prefix + comms.comms[comm_count].name;
        if (comm2 == comm_name) {
          comms.comms[comm_count].out(robot, msg, args);
        }

      }
    }
  }
});

function restart(mess, args) {
  if (!mess.member.hasPermission("ADMINISTRATOR ")) {
    return mess.channel.send("У вас нет прав"); /* Если у исполнителя команды нету привилегии MANGAGE_MESSAGES, он не сможет её использовать */
  }
  if (args[1] == "000") {
    mess.delete().catch(); // Удаление сообщения пользователя после отправки 
    console.log("Bot restarting...");
    mess.channel.send("Перезагрузка бота...").then(process.exit(0));
    mess.channel.send(robot.user.username + " запустился!");
    console.log(robot.user.username + " запустился!");
  } else {

    console.log("Invalid restart password.");
    mess.channel.send("Неверный код автивации.");
  }
  
}

robot.login(token); // Авторизация бота

File: bot.py
from modules import *

bot = commands.Bot(command_prefix='!', help_command=None)
bot.add_cog(functions.Bot(bot))

@bot.event
async def on_ready():
    print('Logged in as:\n{0.user.name}\n{0.user.id}'.format(bot))
    channel = bot.get_channel(int(786919558522994716))
    embed = discord.Embed(colour=0xe6002e, title='sys.INFO', description=f'```yml\n{bot.user.name} запустился!\n```')
    await channel.send(embed=embed)

@bot.event
async def on_command_error(ctx, error):
    if isinstance(error, CommandNotFound):
        #print(bot.user.avatar_url())
        await ctx.send(embed = discord.Embed(colour=0xe6002e, title='sys.INFO', description = f'** {ctx.author.name}, данной команды не существует.\n'
                                                                                              f'Для получения справки по командам введите: ```diff\n!help```**'))
    #raise error

bot.run(config['token'])

'''
from modules import *

client = discord.Client()

@client.event
async def on_ready():
    print(f'We have logged in as {client.user.name}')
    channel = client.get_channel(int(786919558522994716))
    embed = discord.Embed(colour = 0xe6002e, title='sys.INFO', description = f'{client.user.name} запустился!')
    #embed.set_author(name="System")
    #embed.set_author(name="Bot", url="https://yandex.ru/", icon_url="url image ico")
    #embed.set_thumbnail(url="url image ico")
    #embed.add_field(name=" ", value=" ", inline=True)
    #embed.set_footer(text=" ")
    await channel.send(embed = embed)

@client.event
async def on_message(message):
    if message.author == client.user:
        return

    if message.content.strip()[0] != '!':
        return
    
    args = message.content.strip().lstrip(config['prefix']).split(" ")
    
    #try:
    if args[0] in functions.methods:
        function = getattr(functions, args[0])
        await function(client, message, args)
    else:
        return
    #except Exception as err:
    #    print("Error: \n" + str(err))
    #    embed = discord.Embed(colour = 0xe6002e, title=f"sys.ERROR\n {str(err)}")
    #    await message.channel.send(embed = embed)
    #    return

client.run(config['token']) # Обращаемся к словарю settings с ключом token, для получения токена
'''